app/services/apiRequest.js

```diff
@@ -0,0 +1,18 @@
+import fetch from 'isomorphic-fetch';
+import { API_URL } from 'config';
+import parseJwt from './parseJwt';
+
+const apiRequest = async ({ url, method, headers, body }) => {
```

Comment:
Please use simple parameters for this function in the following order: `(method, url, body = {}, headers = {})`.